capella.go

```diff
@@ -126,6 +126,10 @@ func ProtoRequestToCapellaRequest(block *SubmitBlockRequest) *capella.SubmitBloc
 			Address:        b20(withdrawal.Address),
 		})
 	}
+	value, err := uint256.FromHex(block.BidTrace.Value)
+	if err != nil {
+		return nil, err
```

Comment:
may want to mention block bid trace value in the error message instead of just returning `hex string without 0x prefix`